.chloggen/debug-exporter-normal-verbosity-logs.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

```suggestion
change_type: enhancement
```

Same as https://github.com/open-telemetry/opentelemetry-collector/pull/10280#discussion_r1624076976, this is not breaking